sql/04-load-table-student.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO student VALUES (1510, 'Jordan', 9);
```

Comment:
This is a workaround from loading the CSV file directly.  Different database systems have different ways of doing this (copy, load data infile), but it'll be good for you to learn the methods because sometimes you'll end up having to work with big files, or you will be automating things and so you won't want to write the code by hand.